src/lib/common/enums/relationshipType.ts

```diff
@@ -1,15 +1,15 @@
 export enum RelationshipType {
-  Association = 'association',
-  Serving = 'serving',
-  UsedBy = 'usedby',
-  Assignment = 'assignment',
-  Triggering = 'triggering',
-  Access = 'access',
-  Influence = 'influence',
-  Realization = 'realization',
-  Realisation = 'realisation',
-  Specialization = 'specialization',
-  Flow = 'flow',
-  Composition = 'composition',
-  Aggregation = 'aggregation',
+  Association = 'Association',
```

Comment:
Is there a particular reason to not keep the values in lowercase?

Reply:
Hi, @Diorbert.
I made a mistake changing these values to uppercase and fixed it in the branch I'm currently working on. These values refer to the translated relationship type in the model and must be kept in lower case.